Sloth.Core/SlothDbContext.cs

```diff
@@ -84,7 +84,7 @@ public async Task<string> GetNextKfsTrackingNumber(DbTransaction transaction = n
             }
         }
 
-        public async Task<string> GetNextDocumentNumber(DbTransaction transaction = null)
+        public virtual async Task<string> GetNextDocumentNumber(DbTransaction transaction = null)
```

Comment:
virtual was just so the db can be mocked out, correct?

Reply:
Yes.